todo_app/flask_config.py

```diff
@@ -3,6 +3,7 @@
 
 class Config:
     """Base configuration variables."""
+    # SECRET_KEY = "574e0ac08ec210115e288fa9"
```

Comment:
I hope this isn't actually a secret! We should delete the comment either way, but shout if this is secret (e.g. from Trello) since we should also revoke any access from that if so